src/net/sourceforge/kolmafia/utilities/ChoiceUtilities.java

```diff
@@ -31,6 +32,23 @@ public class ChoiceUtilities {
     Pattern.compile("whichchoice=(\\d+)"),
   };
 
+  private static final Pattern[] PARSE_PATTERNS = {FORM_PATTERN, LINK_PATTERN};
+
+  private static final String CHOICE_PHP = "choice.php";
+  public static final String SECRET_CHOICE = "(secret choice)";
```

Comment:
On the other hand, having a public constant which is a KoLmafia-assigned name (rather than KoL-assigned) seems OK. It is used twice in this file.

I'm assuming it's public because you intend to include tests which will validate that that String is included when it is appropriate. :)